packages/core/core/src/ReporterRunner.js

```diff
@@ -41,7 +41,7 @@ export default class ReporterRunner {
     });
 
     if (this.options.patchConsole) {
-      patchConsole();
+      //patchConsole();
```

Comment:
we need to fix this 😢

Reply:
Isn't the option patchConsole: false working?